Core/Core/Planner/CalendarEvent/View/CustomFrequencyComponents/WeekDaysSelectionListView.swift

```diff
@@ -0,0 +1,62 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+struct WeekDaysSelectionListView: View {
```

Comment:
Event if not used in the `var body: some View { .. }` ?
I checked in code whether this is being used in some way, but couldn't figure it out. 
Can you elaborate more about that ?

Reply:
I think we rarely (if ever) need to use the actual value. The point is, that since this is an `@Environment` property, when it changes (when the user changes text size) the view which contains it will be redrawn, updating the text size instantly.